### PR TITLE
create orders

### PR DESCRIPTION
**Pull request by manhbld-0962**
==> Tittle: create order

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/18310

==> Attachment File:

### CODE REVIEW

**Comment by manhbld-0962** [comment]
ready

File: app/models/order.rb [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
   belongs_to :booking
 
+  delegate :name, to: :food, prefix: :food
+
   validates_uniqueness_of :booking_id, scope: :food_id
   validates :food_id, presence: true, on: :create
   validates :number, presence: true, numericality: {only_integer: true}
+
+  def food_name_titleize
+    self.food_name.titleize
```
Comment (hungct-1121) [inline app/models/order.rb]: thêm 1 cái `delegate :titleize, to :food_name, prefix: true` là khỏi cần viết method :v

**Comment by manhbld-0962** [comment]
ready

File: app/views/orders/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+<div class="title-name">
+  <span class="content-name"><%= t("orders.index.name_page") + ": #{@booking.user.titleize_name}" %></span>
+</div>
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <span class="text-center"><%= t(".text_status") %>
+      <span class="status-booking"><%= @booking.checkout.upcase %></span>
+      <% if @booking.unpaid? %>
+        <% if current_user.admin? %>
```
Comment (hieplt-0914) [inline app/views/orders/index.html.erb]: Sao ko gop 2 cau dieu kien vao lam 1 ?

**Comment by manhbld-0962** [comment]
ready

**Pull request merged**